packages/react-menu/src/components/MenuGroup/useMenuGroup.ts

```diff
@@ -13,11 +13,11 @@ const mergeProps = makeMergePropsCompat<MenuGroupState>({ deepMerge: menuGroupSh
 /**
  * Given user props, returns state and render function for a MenuGroup.
  */
-export const useMenuGroup = (
+export const useMenuGroup: (
```

Comment:
Was there something breaking with the previous version just typing props ?

Reply:
It's an issue with `import()` and API extractor, I used `function` and simplified types there